### ISSUE TITLE
Reduce SemanticsTester noise in goldens

### ISSUE THREAD

**Issue by CaseyHillers** [flutter/flutter#115471]
This is a list of fixes we can add to the framework semantic tester to reduce the noise in goldens

1. Change all node colors to a single color (black, blue, etc). If a hidden node is inserted to the tree, all nodes are impacted 
2. Lower case all text in the nodes. Casing shouldn't matter, and for changes like Material 2 -> 3 going from SCREAMING_CAPS to Sentence structure, it adds noise

**Comment by CaseyHillers** [comment]
\cc @gspencergoog @dnfield

**Comment by dnfield** [comment]
To be clear, the _real_ fix is to not use the semantics tester this way, but yes these seem like good ideas.

One team did suggest that removing different colors would make it harder to see relationships. I am .. unsympathetic. They should have other ways to diagnose this if they want.

**Comment by dnfield** [comment]
@pdblasi-google fyi

**Comment by CaseyHillers** [comment]
> One team did suggest that removing different colors would make it harder to see relationships.

@dnfield I believe that one team also implements their own semantics tester, so this would affect everyone but them.

**Comment by jonahwilliams** [comment]
another possibility is to make the output of the semantics tester _so_ unstable that it is completely impossible to use in scubas.

**Comment by pdblasi-google** [comment]
I mean, if we're going to go _that_ route I'd just do an assert that there's no `SemanticsDebugger` in the tree when we try to do a golden comparison. I'd venture to guess that'd make some people unhappy and/or drive them to try to make their own `SemanticsDebugger`...
